packages/airnode-deployer/terraform/aws/templates/oevGw.yaml.tpl

```diff
@@ -0,0 +1,191 @@
+openapi: "3.0.2"
```

Comment:
How and where is this generated?

Reply:
It's not generated, it was written manually.